.prettierignore

```diff
@@ -28,33 +24,17 @@ change
 __snapshots__
 
 # package specific files
+*.hbs
```

Comment:
handlebars is supported in prettier 2.3 - tested that out didn;t work as expected. so I added it here for future version bump (wont be triggered on hbs files in vscode etc )

Reply:
I'm guessing it didn't work because it doesn't respect the language within the file? If so it would probably be good to add back the previous comment about why it's excluded.